src/GafferImage/Display.cpp

```diff
@@ -268,42 +299,54 @@ class GafferDisplayDriver : public IECoreImage::DisplayDriver
 			Display::driverCreatedSignal()( driver.get(), parameters.get() );
 		}
 
-		ConstFloatVectorDataPtr getTile( const V2i &tileOrigin, size_t channelIndex )
+		struct TileCacheElement
```

Comment:
Just `Tile` seems as descriptive as `TileCacheElement`, and the symmetry of `m_tiles` holding `Tile` would be nice.